src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public abstract class Machine {
+
+    public abstract void doWork();
+
+    public abstract void stopWork();
+
+}
+

```

Comment:
redundant empty lines, should be just 1 at the end of the file
```suggestion
}

```